### DISCUSSION TITLE
desktop-settings 11.8.6 does not replace chrome-flags symlink with actual file

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#328] [Q&A] [answered]
If I understand correctly, [desktop-settings release 11.8.6](https://github.com/manjaro-sway/desktop-settings/compare/11.8.5...11.8.6) replaces the symlink `chrome-flags.conf -> electron-flags.conf` with an actual file - to fix #322.

But running the recommended `cp -rf /etc/skel/.config/* ~/.config` after the update did not replace the existing symlink. I had to manually remove the symlink and run `cp -rf ...` again.

Probably not a huge issue but it might easily be missed, and may cause problems if chromium browsers expect the flags in one line now while electron expects multiple lines - see https://github.com/manjaro-sway/manjaro-sway/issues/322#issuecomment-1245278892

**Comment by boredland** [comment]
Ha, I always removed them manually, that's why I never encountered that. Perhaps we should invest a bit in a more elaborate "copy skel from upstream" command. Ideas?

**Reply by boredland** [reply]
this is how arcolinux does it. guess we could just copy that?
https://github.com/arcolinux/arcolinux-on-the-road/blob/cfbcc902b9520cc4ff73584dd80f34c54a158c75/root/usr/local/bin/skel

**Reply by boredland** [reply]
I just tried to integrate this. It is available as "skel" after the latest update. Seems to work just fine. Enjoy!

**Comment by boredland** [comment, marked as answer]
just use the new `skel` script on the command line!
